epsagon/trace.py

```diff
@@ -35,6 +36,7 @@
 DEFAULT_MAX_TRACE_SIZE_BYTES = 64 * (2 ** 10)
 MAX_METADATA_FIELD_SIZE_LIMIT = 1024 * 3
 FAILED_TO_SERIALIZE_MESSAGE = 'Failed to serialize returned object to JSON'
+IS_PY_VERSION_ABOVE_3_6 = sys.version_info[0] == 3 and sys.version_info[1] >= 7
```

Comment:
add a comment explaining its above (> and not >=) 3.6 so it will be clear it shouldn't include 3.6 in the condition